api/base/settings/defaults.py

```diff
@@ -151,6 +151,7 @@
         '2.1',
         '2.2',
         '2.3',
+        '2.4',
```

Comment:
Nope. Although this reminds me that the versioning tests really need to be revisited...